<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewPage</class>
 <widget class="QWidget" name="OverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelAlerts">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop:0 #F0D0A0, stop:1 #F8D488); color:#000000;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="frameDarksend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>142</y>
     <width>750</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; border-radius: 6px; border: 1px solid white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="darksendStatus">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>59</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>(Last Message)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleDarksend">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white; background: transparent; color: white;</string>
    </property>
    <property name="text">
     <string> Start / Stop Mixing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); border: none;</string>
    </property>
    <property name="text">
     <string>Darksend Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnonymizedText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>Darksend Balance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>74</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>Completion:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="darksendProgress">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>154</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnonymized">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>0 BCR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>109</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>Amount / Rounds:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAmountRounds">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>0 BCR / 0 Rounds</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>Submitted Denom:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSubmittedDenom">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="statusTip">
     <string>The denominations you submitted to the Basenode. To mix, other users must submit the exact same denominations.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
   </widget>
   <widget class="QLabel" name="darksendEnabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>105</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);border: none;</string>
    </property>
    <property name="text">
     <string>Enabled/Disabled</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>17</y>
     <width>750</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 6px; border: 1px solid white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white; border: none;</string>
    </property>
    <property name="text">
     <string>Available Balance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnconfirmed">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>291</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Total of transactions that have yet to be confirmed, and do not yet count toward the current balance</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white; border: none;</string>
    </property>
    <property name="text">
     <string notr="true">0 BCR</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelBalance">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>9</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your current balance&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white;border: none;</string>
    </property>
    <property name="text">
     <string notr="true">0 BCR</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>83</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white; border: none;</string>
    </property>
    <property name="text">
     <string>Unconfirmed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImmatureText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white; border: none;</string>
    </property>
    <property name="text">
     <string>Immature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImmature">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>311</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Mined balance that has not yet matured</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white; border: none;</string>
    </property>
    <property name="text">
     <string notr="true">0 BCR</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>267</y>
     <width>750</width>
     <height>225</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; border-radius: 6px; border: 1px solid white; color: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: white;border: none;</string>
        </property>
        <property name="text">
         <string>&lt;b&gt;Recent transactions&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listTransactions">
      <property name="styleSheet">
       <string notr="true">background: transparent; border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>labelAlerts</zorder>
  <zorder>frameDarksend</zorder>
 </widget>
 <resources>
  <include location="../bitcredit.qrc"/>
 </resources>
 <connections/>
</ui>
